### DISCUSSION TITLE
<q-img/> GET http://localhost:8080/path/file.ext 404 (Not Found)

### DISCUSSION THREAD

**Discussion by JozefPlata** [quasarframework/quasar#11831] [General - Components / Directives / etc]
Hi, 

So again I'm stuck on resolving paths. I'm still on local development. At first I thought that's because I'm loading it from the wrong context, but now I've been testing it on a `<q-img/>` and it's probably a path resolving issue with local development.

So this doesn't resolve the file, even the path is correct. It's the default path for the quasar icons.

```
<template>
  <q-img :src="img"/>
</template>

<script>
import { defineComponent, ref } from 'vue';

export default defineComponent ({
  data() {
    return {
      img: "public/icons/favicon-128x128.png"
    }
  }
})
</script>
```

Ofcourse this works just fine:

`<q-img src="~/public/icons/favicon-128x128.png"/>`

However the whole point is to resolve the path in `<script>` section. So what am I doing wrong?

**Comment by jj-fx** [comment]
Ok I got it finally, it should be:

```
    return {
      img: "icons/favicon-128x128.png"
    }
```

**Comment by rstoenescu** [comment]
There is no /public/... unless you have a public folder in the public folder and  your publicPath is the default one (/) :).

Use no forward slash at the beginning and directly reference the resource name. Please read the handling resources page in docs. It's part of the magic of q/app and by reading that page it'll make a lot of sense.
